contracts/Admin/VBNBAdmin.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+pragma solidity 0.8.13;
+
+import "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+import "./VBNBAdminStorage.sol";
+
+contract VBNBAdmin is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable, AccessControlledV8, VBNBAdminStorage {
+    using SafeERC20Upgradeable for IWBNB;
+
+    /// @notice Emitted when PSR is updated
+    event ProtocolShareReserveUpdated(
+        IProtocolShareReserve indexed oldProtocolShareReserve,
+        IProtocolShareReserve indexed newProtocolShareReserve
+    );
+
+    function initialize(
+        VTokenInterface _vBNB,
+        IProtocolShareReserve _protocolShareReserve,
+        IWBNB _WBNB,
+        address _comptroller,
+        address accessControlManager
+    ) external initializer {
+        vBNB = _vBNB;
+        protocolShareReserve = _protocolShareReserve;
+        WBNB = _WBNB;
+        comptroller = _comptroller;
```

Comment:
Would it make sense to assume that the comptroller will be `vBNB.comptroller()`? This way we can save a storage slot